content/lua/concepts/tables/terms/insert.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'insert()'
+Description: 'Inserts an element to an existing table at a specified index.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Tables'
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`.insert()`** method inserts a value in a table.
+
+## Syntax
+
+The `value` can be inserted at a specific `index`.  
+
+```lua
+table.insert(table, index, value)
+```
+
+If `index` is not provided, `value` is inserted at the end of `table`. It returns `table`. Stacks, queues, and double queues can be implemented in this way.    
+
+## Codebyte Example
```

Comment:
```suggestion
## Example
```
We don't have codebytes implemented for lua.

Reply:
Implemented suggested updates